repository/r.json

```diff
@@ -461,6 +461,17 @@
 				}
 			]
 		},
+		{
+			"name": "RBS Syntax",
```

Comment:
We try to have language packages simply named after the language, without suffixes like "syntax" (or "highlighting" or "language" etc)

Reply:
```suggestion
			"name": "RBS",
```